llumnix/queue/utils.py

```diff
@@ -19,7 +24,46 @@
 from llumnix.queue.ray_queue_client import RayQueueClient
 from llumnix.queue.zmq_utils import get_open_zmq_ipc_path
 from llumnix.queue.queue_type import QueueType
+from llumnix.logger import init_logger
+
+logger = init_logger(__name__)
+
+class AsyncPutQueueActor:
```

Comment:
why put this actor here?

Reply:
put it in llm_engine.py.
leave this next pr to refine